integration/tolerances/tolerance_default.json

```diff
@@ -0,0 +1,3 @@
+{
+    "default": 0.0001
```

Comment:
I preferred this the other way where it was just a string sent in and not a file. It is more clear from a debugging stand-point by looking at the command that was being run. Unless @jamiebramwell has advice on the tolerances being different for each field.

Reply:
Both the string query and file are working, but if you want the file option completely removed for simplicity I can do that as well.

The `tolerace_default.json` specifically was just to show how to set things up.

Reply:
Don't remove the option of using a file.  I can definitely see the use of that.  I would just prefer the simpler singular value to be used in the default case.

Reply:
You can remove this file now since it isnt being used anywhere.